devices/lidl/hg08673.json

```diff
@@ -175,6 +175,14 @@
         },
         {
           "name": "state/consumption"
+          "refresh.interval": 300,
```

Comment:
The refresh interval should be roughly 60 seconds larger than the max reporting interval of the attirbute. You have defined 360 max reporting, so it should be 420 up here.

As a side note, the vast majority of devices uses a max reporting interval of 300 secs and a respective refresh interval of 360.